### PR TITLE
Suggested modifs on the paper

### PR DESCRIPTION
**Pull request by jcohenadad**
Easier to read and more consistent with the "43" below

### CODE REVIEW

**Comment by jcohenadad** [comment]
I'm not sure how to cross-ref a line (@agahkarakuzu do you know?) so I'm copy/pasting here:
image

unclear what a "perfect" ellipse is. When it is not a circle (ie: different from zero), then it is an elllipse. So what makes it perfect?

**Comment by jcohenadad** [comment]
I would move (and merge) that whole section under 2.1:

image

**Comment by jcohenadad** [comment]
I would move this:
image

There:
image

**Comment by agahkarakuzu** [comment]
@jcohenadad there's a GitHub app called [Review Notebook](https://github.com/marketplace/review-notebook-app/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW4yMDIz#plan-Free), installing it makes it somewhat easier. Otherwise the only option is https://github.com/valosekj/PAM50-normalized-metrics-paper/blob/41fc3d7505f4cebe13afff122425889d3e950a10/content/index.ipynb#L158 from code view.

**Comment by jcohenadad** [comment]
weird:

image

**Comment by agahkarakuzu** [comment]
@jcohenadad that's how notebook links to the multiple occurrences of a citation. There are [different styles](https://jupyterbook.org/en/stable/content/citations.html#select-your-bibliography-style) available.  I agree that the same numeric format repeated for both parent/children references may be confusing. For example https://oreoni.github.io/01/introduction.html uses 

image

File: content/index.ipynb [hunk 1]
```diff
@@ -168,7 +168,7 @@
...
+    "Statistical analysis was conducted using the SciPy Python library `v1.10.1` {cite:p}`Virtanen2020-hl`. Descriptive statistics, including mean and standard deviation, were computed for age, sex, height, and weight. The Shapiro-Wilk normality test was used to assess data normality. Differences between males and females in age, height, and weight were examined using the Wilcoxon rank-sum test. Morphometric measures in PAM50 space were averaged across participants for each slice and compared between sex and MRI vendors using the Wilcoxon rank-sum test. The significance level was set to `alpha = 0.001`.\n",
```
Comment (valosekj) [inline content/index.ipynb]: For sex, we report only the subject count. Not mean and std.
image

**Comment by valosekj** [comment]
> unclear what a "perfect" ellipse is. When it is not a circle (ie: different from zero), then it is an elllipse. So what makes it perfect?

Agree, I removed this sentence in https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/4/commits/92db4c39ace32f335f5b666a7bb972ee10f6520a.

File: content/index.ipynb (hunk 1 above)
Comment (valosekj) [reply content/index.ipynb]: `sex` removed in https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/4/commits/7ef49d20ff28111709a7c4ad1ea15a04050d29c8

**Comment by valosekj** [comment]
@jcohenadad, thanks for your feedback! I addressed your suggestions in my recent commits. If you agree, we can merge this PR.

**Pull request merged**